src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -234,6 +234,22 @@ def add_records(
     """
     url = "/api/v1/datasets/{id}/records".format(id=id)
 
+    for record in records:
+        cleaned_responses = []
+        response_without_user_id = False
+        for response in record["responses"]:
+            if response["user_id"] is None:
+                if response_without_user_id is True:
+                    warnings.warn(
+                        f"Multiple responses without `user_id` found in record {record}, so just the first one will be used while the rest will be ignored."
+                    )
+                    continue
+                else:
+                    response["user_id"] = client.get("api/me").json()["id"]
+                    response_without_user_id = True
```

Comment:
```suggestion
            if response["user_id"] is None and response_without_user_id:
                warnings.warn(
                    f"Multiple responses without `user_id` found in record {record}, so just the first one will be used while the rest will be ignored."
                )
                continue
            else:
                response["user_id"] = client.get("api/me").json()["id"]
                response_without_user_id = True
```

Reply:
I'll apply the boolean change you mentioned above, but not this one, as we only need to retrieve the `user_id` from `/api/me` when the `user_id` is None, if it has any value, then we just keep it. It will fail anyway if the `user_id` doesn't exist in Argilla.